tests/test_convert_lit_checkpoint.py

```diff
@@ -211,10 +211,51 @@ def test_check_conversion_supported_adapter_v2():
         check_conversion_supported(lit_weights=lit_weights)
 
 
-def test_check_conversion_supported_lora():
-    from scripts.convert_lit_checkpoint import check_conversion_supported
-
-    lit_weights = {"some.key.name": "some.key.value", "error.key.lora": "some.key.value"}
-
-    with pytest.raises(ValueError, match=r"Model weights must be merged using"):
-        check_conversion_supported(lit_weights=lit_weights)
+def test_merge_lora_weights_for_conversion(tmp_path):
```

Comment:
Simply unit-testing `merge_lora_weights_for_conversion` would be enough to test the changes in this PR. This test is unnecessarily complex, it doesn't need to train a model